pkg/rulefmt/rulefmt.go

```diff
@@ -155,8 +155,8 @@ func testTemplateParsing(rl *Rule) (errs []error) {
 	}
 
 	// Trying to parse templates.
-	tmplData := template.AlertTemplateData(make(map[string]string), 0)
-	defs := "{{$labels := .Labels}}{{$value := .Value}}"
+	tmplData := template.AlertTemplateData(make(map[string]string), make(map[string]string), 0)
+	defs := "{{$labels := .Labels}}{{$external_labels := .ExternalLabels}}{{$value := .Value}}"
```

Comment:
It feels odd to have Go style names for fields, and Javascript style names for variables.

Reply:
My first choice was `$externalLabels` but I felt it looked bad.

Reply:
What do you propose ?

Reply:
externalLabels would be my thought.

Reply:
Done.